yapftests/reformatter_basic_test.py

```diff
@@ -421,8 +510,8 @@ def testSingleComment(self):
 
   def testCommentsWithTrailingSpaces(self):
     unformatted_code = textwrap.dedent("""\
-        # Thing 1    
```

Comment:
I think this change was unintended here (and might defeat the purpose of this test)

Reply:
yes. seems like my IDE lacks the context of whats happening here.

Reply:
fixed in #857